### DISCUSSION TITLE
some settings in pro active invitation have no effect?

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1849] [General]
when i check these options in pro active invitation they are not respected in the new widget.
![image](https://user-images.githubusercontent.com/13809377/207594776-e4df2961-4339-45c7-9466-757fd3803b33.png)

it seems that these settings can only be specified here:
![image](https://user-images.githubusercontent.com/13809377/207595432-1609f3ad-7cc4-41ca-afb8-05adaaf45309.png)

**Comment by remdex** [comment]
These required things are only for an old widget, but not for a new.

**Comment by comsyspro** [comment]
thank you for clarification. would it be possible to add a hint to the settings which are only supported for the old and new widget version?
